### ISSUE TITLE
Syntax error in type annotations

### ISSUE THREAD

**Issue by pmeier** [pytorch/vision#2061]
In the light of [first inline type annotations](/pytorch/vision/pull/2034), a static type check via `mypy` was requested in https://github.com/pytorch/vision/pull/2034#pullrequestreview-385008031.
Running `mypy` on the current `torchvision` results in syntax errors in several type annotations. For example:

```
torchvision/io/_video_opt.py:86: error: syntax error in type comment '(List[int])'
```

Since syntax errors are [fatal](https://mypy.readthedocs.io/en/stable/error_code_list.html#report-syntax-errors-syntax) they cannot be silenced even with [`ignore_errors`](https://mypy.readthedocs.io/en/stable/config_file.html#suppressing-errors) set. Furthermore, `mypy` stops the checking after it encountered a module with syntax errors. 

Thus, if we want to include `mypy` checks in CI and test if they work we need to either fix them or disable them temporarily and fix them later. Since they seem to be related to `torch.jit`, I have no idea how much work fixing them would be or how much damage temporarily disabling them will do.

**Comment by fmassa** [comment]
I've noticed similar issues as well in the past, and I tried to fix some of those in https://github.com/pytorch/vision/pull/1696

We need those type annotations as those functions are being used with TorchScript, so we need to find a way to make them compliant with TorchScript and mypy.

cc @eellison for thoughts.

**Comment by pmeier** [comment]
You never merged the PR although @eellison approved it. Does that mean the way you did it does not work with TorchScript? If adding the return type as you did does indeed work with TorchScript could you rebase and merge your PR. I would take up the rest.

**Comment by fmassa** [comment]
That PR had failing tests, and I never had the chance to get back to it and fix it so that it is compatible with mypy and torchscript.

@zhangguanheng66 are you still looking at fixing type annotations in #1696?

**Comment by zhangguanheng66** [comment]
I think the the output annotation `Tuple[Tensor, Tensor]` is not supported by torchscript. @eellison

**Comment by pmeier** [comment]
This is fixed by #1696 and #2195.
